Source/RunActivity/RunActivity.csproj

```diff
@@ -49,6 +49,10 @@
     <Reference Include="EmbedIO, Version=3.4.3.0, Culture=neutral, PublicKeyToken=5e5f048b6e04267e, processorArchitecture=MSIL">
       <HintPath>..\packages\EmbedIO.3.4.3\lib\netstandard2.0\EmbedIO.dll</HintPath>
     </Reference>
+    <Reference Include="glTFLoader, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\glTF2Loader.1.0.0\lib\net452\glTFLoader.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
     <Reference Include="GNU.Gettext, Version=1.1.5151.39896, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
When I check the .csproj file: https://github.com/KhronosGroup/glTF-CSharp-Loader/blob/master/glTFLoader/glTFLoader.csproj
I see: `<TargetFrameworks>netstandard1.3;net35</TargetFrameworks>`
netstandard1.3 should be fine for net5/6. It is autogenerated directly from the original schema. The only thing I fear from is the "extras", I found at least one gltf that crashes it because of some "extras" entries.